gatsby-node.js

```diff
@@ -32,6 +33,24 @@ exports.createPages = async ({ graphql, actions: { createPage } }) => {
     }
   `)
 
+  paginate({
+    createPage,
+    items: pages.data.allProducts.edges,
+    itemsPerPage: 6,
```

Comment:
Should be bump this up to a higher default? 12?

We could also set this as a `GATSBY_ITEMS_PER_PAGE` `ENV` perhaps?

```js
itemsPerPage: process.env.GATSBY_ITEMS_PER_PAGE || 12
```

Thoughts?

Reply:
@notrab I only set it to `6` to actually get some pagination, seeing as there's only 12 products in the demo store

Reply:
Makes total sense, for others though, this might be a bit overkill.

Should we set this via `ENV`, place it inside the `netlify.toml` which allows anyone to define it on deploy?

We could also fallback to `9` by default?

Reply:
I've gone ahead and added `GATSBY_ITEMS_PER_PAGE`. Might be worth using `parseInt` on the value, just to be sure.